cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -1460,7 +1483,8 @@ toAlonzoPParams ProtocolParameters {
                                (Ledger.boundRational protocolParamTreasuryCut)
 
       -- New params in Alonzo:
-    , Alonzo._coinsPerUTxOWord  = toShelleyLovelace utxoCostPerWord
+    , Alonzo._coinsPerUTxOWord  = -- adapted from babbage protocol parameter field
```

Comment:
Why the comment? I don't understand.

Reply:
Left over comment from when I previously did the byte to work conversion.  Deleting.